frontend/tests/playwright/product-list/filters.spec.ts

```diff
@@ -98,17 +111,7 @@ test.describe('product list filters', () => {
             'data-value'
          );
          await checkRefinementValue(secondFacetOptionValue, page);
-
-         // Remove the newest refinement.
-         await page
-            .getByTestId(`current-refinement-${secondFacetOptionValue}`)
-            .getByRole('button', { name: /remove/i })
-            .click();
-
-         // Check that the refinement value is not in the current refinements.
-         await expect(
-            page.getByTestId(`current-refinement-${secondFacetOptionValue}`)
-         ).not.toBeVisible();
+         await removeAndCheckRefinement(secondFacetOptionValue, "remove", page);
```

Comment:
We can use `!` to assert that `secondFacetOptionValue` is not null

```suggestion
         await removeAndCheckRefinement(secondFacetOptionValue!, "remove", page);
```

Reply:
Thanks @ardelato! Updated in https://github.com/iFixit/react-commerce/pull/1244/commits/5e04f657ae3ee3e9e5d10ec3ef1fad2784e30af2.